eslzArm/README.md

```diff
@@ -0,0 +1,385 @@
+# Enterprise-Scale Landing Zones ARM templates
+
+This folder contains the first-party ARM templates for Enterprise-Scale which and are being used when deploying and bootstrapping in the Azure Portal, which is our recommendation as it will 1) save you tremendous amount of time, 2) accelerate your journey, and 3) optionally bootstrap your GitHub repository with ready-to-use ARM templates if you want to pivot to infrastructure-as-code post deployment.
+
+For customers who cannot deploy via portal, but rather want to clone the repository and sequence the deployments on their own using the same ARM templates, they can follow the manual deployment instructions below.
+
+> **Note:** There's a strict sequencing required in order to achieve the same outcome as when deploying via the Azure portal, and any modification and changes to the templates are not supported.
+
+## Do-It-Yourself deployment instructions for Enterprise-Scale using Azure PowerShell
+
+Prerequisites:
+
+* [Azure PowerShell module](https://docs.microsoft.com/powershell/azure/install-az-ps?view=azps-6.3.0)
+* [Sign in and get started](https://docs.microsoft.com/powershell/azure/get-started-azureps?view=azps-6.3.0#sign-in-to-azure)
+* [How to clone a GitHub repository](https://docs.github.com/github/creating-cloning-and-archiving-repositories/cloning-a-repository-from-github/cloning-a-repository)
+
+````powershell
+
+# Do-It-Yourself instructions for doing Enterprise-Scale
+
+# Change the variables below to contain the right values for your tenant, subscription, address space etc.
+
+$ESLZPrefix = "ESLZ"
+$Location = "westeurope"
+$DeploymentName = "EntScale"
+$TenantRootGroupId = (Get-AzTenant).Id
+$ManagementSubscriptionId = "<replace me>"
+$ConnectivitySubscriptionId = "<replace me>"
+$ConnectivityAddressPrefix = "<replace me>"
+$IdentitySubscriptionId = "<replace me>"
+$SecurityContactEmailAddress = "<replace@this.address>"
+$CorpConnectedLandingZone = "<replace me>"
```

Comment:
```suggestion
$CorpConnectedLandingZone = "<replace me>" # This should be a Subscription ID for a Subscription you wish to be moved to the "Corp" Management Group
```

Added this comment as it wasn't clear as to what was expected here to input. Also do we need to change the "connected" wording here as technically it wont be connected to the Hub (networking wise)?